source/atlas-cli-deploy-docker.txt

```diff
@@ -432,3 +439,55 @@ directory:
             sudo dpkg -i mongosh.deb
             mongosh --version
         - run: mongosh 'mongodb://localhost/?directConnection=true' --eval 'show dbs'
+
+
+.. _convert-to-local-dev:
```

Comment:
It would be nice to link to the docker hub links, the `mongo:8.0` links to https://hub.docker.com/_/mongo and our image to https://hub.docker.com/r/mongodb/mongodb-atlas-local